client/src/screens/schedules/Detail.js

```diff
@@ -326,7 +331,14 @@ class Detail extends Component {
             />
           </Holder>
         </Modal>
-        <ScrollView>
+        <ScrollView
+          refreshControl={
+            <RefreshControl
```

Comment:
TIL you can override this. But we've never needed to do this before, are you sure it's necessary? Or have we always just done it entirely wrong? I think Tim's suggestion is sound though - we should be able to use the apollo loading status to avoid needing state.

Reply:
This is a stop gap until we have proper subscriptions (people submitting availability on web, being able to easy see it on mobile without going back to home screen for refresh). Thought of it as an extra feature. 

I can switch it to use the loading status though.